gateway-service/src/main/java/com/ca/mfaas/gateway/ws/WebSocketProxyServerHandler.java

```diff
@@ -66,7 +73,7 @@ public void addRoutedServices(String serviceId, RoutedServices routedServices) {
     public void afterConnectionEstablished(WebSocketSession webSocketSession) throws Exception {
         URI uri = webSocketSession.getUri();
 
-        String[] uriParts = uri.getPath().split("/", 5);
+        String[] uriParts = uri.getPath().split(SEPARATOR, 5);
```

Comment:
Hello Reet,
Thanks for this fixing. Just one suggestion. Could you, please, add this check on null: `uri != null && uri.getPath() != null`?
```suggestion
        String[] uriParts = null;
        if (uri != null && uri.getPath() != null) {
            uriParts = uri.getPath().split(SEPARATOR, 5);
        }
```

Reply:
Done :)